README.md

```diff
@@ -67,15 +67,33 @@ make sky130hd_temp
+
+  Block Architecture:
+   - Temperature-sensitive ring oscillator and stacked zero-VT devices.
+    ![plot](./markdown%20images/tempSensor-BA.png)
+     Documentation **[link](https://fasoc.engin.umich.edu/thermal-sensor/)**
+
+
+
+**[ldo-gen:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/ldo-gen)** The main idea behind a Digital LDO is the use of an array of small power transistors that operate as switches . The use of power transistors as switches facilitates low VDD power management and process scalability which makes Digital LDOs a good potential candidate for power management as we go to lower nodes. With the “Unit Power Switch” as the auxiliary cell, an automatic LDO design tool “LDO_GEN” is developed as part of this project.
+
+  Block Architecture:
+   - Synchronous Digital LDO with optional stochastic flash ADC.
+    ![plot](./markdown%20images/LDO-BA.png)
+    Documentation **[link](https://fasoc.engin.umich.edu/digital-ldo/)**
+
+**[dcdc-gen:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/dcdc-gen)** For synthesizable on-chip power management  circuits, we use the “2:1 SC Cell” auxiliary cell for implementing a switched-capacitor (SC) based DC-DC converter. By varying the number of auxiliary cells, we can achieve wide range of conversion ratios with fine grain resolution. It operates similarly to  a successive approximation analog to digital converter (SAR ADC). Furthermore, since the total structure is simply composed of auxiliary cells, it is ideal for the proposed flow of automating the analog block design.  Documentation **[link](https://fasoc.engin.umich.edu/dc-dc-converter/)**
 
-**[temp-sense-gen:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/dcdc-gen)** A fully automated SoC generator that uses an all-digital temperature sensor architecture, that relies on a new subthreshold oscillator (achieved using the auxiliary cell “Header Cell“) for realizing synthesizable thermal sensors.
 **[cryo-gen:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/cryo-gen)**
-**[dcdc-gen:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/dcdc-gen)**
-**[gdsfactory:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/dcdc-gen)**
-**[lc-dco:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/dcdc-gen)**
-**[ido-gen:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/dcdc-gen)**
-**[scpa-gen:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/dcdc-gen)**
+
+**[gdsfactory:-](https://github.com/idea-fasoc/OpenFASOC/tree/main/openfasoc/generators/gdsfactory)**
```

Comment:
There is some documentation for these structures.

Reply:
I couldn't find documentation in https://fasoc.engin.umich.edu/ for the other four generators is there some other documentations I could look for?

Reply:
Check on slack with Tuohang.